README.demo.md

```diff
@@ -0,0 +1,31 @@
+# DELA nodes
+
+1. `git clone https://github.com/dedis/d-voting.git`
+2. `git checkout demo-20230912`
+3. `cd docker-compose`
+4. `export COMPOSE_FILE=docker-compose.dela.yml`
+5.
+
+```
+cat << EOF > .env
+PUBLIC_KEY=7a97bfc968c74fbf0553fdec1a97d8265bd3a32e31edd61e296ea701a009147e
+PROXYPORT=8080
+HOSTNAME=XXX
+EOF
+```
+6. `docker compose up -d`
+8. get `TOKEN_ARGS_WORKER_{0,1}` from DELA leader and
```

Comment:
```suggestion
7. get `TOKEN_ARGS_WORKER_{0,1}` from DELA leader and
```

Reply:
oh, I'm sorry, that file is not supposed to be in the PR!